aten/src/ATen/FunctionalizeFallbackKernel.cpp

```diff
@@ -315,8 +315,17 @@ static at::Tensor _unsafe_view_functionalize(const at::Tensor & self, at::SymInt
   // See  Note [Propagating strides in the functionalization pass]
   // (for _unsafe_view, I'm just manually doing the shape inference rule here instead of calling the meta function for unsafe_view)
   auto inferred_size = at::infer_size_dv(size, self.sym_numel());
+
   auto stride = at::detail::computeStride(self.sym_sizes(), self.sym_strides(), inferred_size);
-  TORCH_INTERNAL_ASSERT(stride.has_value());
+  if (! stride.has_value()){
+    // See if the view is valid. If it's not, then we copy.
+    // It's OK to copy, because _unsafe_view(x) guarantees that x isn't used
+    // anymore.
```

Comment:
as a reader of this code without much context this comment is not very helpful

1) _unsafe_view doesn't even seem to be used here?
2) are you saying if a stride has a value it implicitly means the view is valid?
3) you say then we copy, is that via the contiguous call?

I think a little bit more precision is required here

Reply:
addressed